### PR TITLE
V6 0x97 appendix v cryptography

### PR DESCRIPTION
**Pull request by danielcuthbert**
Removed all tech detail from V6 into new Appendix. 
Resolved duplicates and issues

### CODE REVIEW

**Comment by tghosth** [comment]
@danielcuthbert please can you just confirm you are comfortable with my tweaks:
https://github.com/OWASP/ASVS/pull/2306/files/781537203fced8a44e8fa30d40b8a6a0fa7c07fa..58fc4ba466231d09584a3f28d544b4e8996fd07f

File: 5.0/en/0x14-V6-Cryptography.md [hunk 1]
```diff
@@ -62,10 +62,9 @@ Cryptographically secure Pseudo-random Number Generation (CSPRNG) is incredibly
 
 | # | Description | L1 | L2 | L3 | CWE |
 | :---: | :--- | :---: | :---: | :---: | :---: |
-| **6.3.1** | [MODIFIED] All random numbers and strings intended to be non-guessable must be generated using a CSPRNG. | ✓ | ✓ | ✓ | 338 |
-| **6.3.2** | 6.3.2 [MODIFIED] Verify that GUIDs are created with an implementation of the GUID v4 or v7 algorithms which utilizes a CSPRNG.                                               |     |  ✓  |  ✓  | 338 |
-| **6.3.3** | Random number generation must work properly under heavy system load, or the system must degrade gracefully. | ✓ | ✓ | ✓ | 338 |
-
+| **6.3.1** | [GRAMMAR, LEVEL L2 > L1] Verify that all random numbers and strings which are intended to be non-guessable must be generated using a cryptographically-secure pseudo-random number generator (CSPRNG). | ✓ | ✓ | ✓ | 338 |
+| **6.3.2** | [MODIFIED] Verify that GUIDs are created with an implementation of the GUID v4 or v7 algorithms which utilizes a cryptographically-secure pseudo-random number generator (CSPRNG). | | ✓ | ✓ | 338 |
```
Comment (randomstuff) [inline 5.0/en/0x14-V6-Cryptography.md:65]: Shouldn't we use UUID here?

Comment (danielcuthbert) [reply 5.0/en/0x14-V6-Cryptography.md:65]: GUID is often used synonymously with UUID but is historically tied to use within a Microsoft world. If we were to be pedantic, while GUID is functionally equivalent in many respects, UUID is the term formally defined by the broader industry standard (RFC 4122), which is why maybe using UUID is generally recommended for clarity and consistency. 

I'm easy either way here

File: 5.0/en/0x14-V6-Cryptography.md [hunk 2]
```diff
@@ -32,23 +49,21 @@ Although this section is not easily penetration tested, developers should consid
...
```
Comment (danielcuthbert) [inline 5.0/en/0x14-V6-Cryptography.md:65]: One question about the use of UUIDs over GUIDs as per RC 4122, I'm easy and if it fits into the general style and approach, happy to adapt UUIDs

**Review by danielcuthbert — COMMENTED** [review]
Happy with these tweaks, see my comment about GUID/UUID and ill leave it to you/elar to make a final call

File: 5.0/en/0x14-V6-Cryptography.md (hunk 1 above)
Comment (tghosth) [reply 5.0/en/0x14-V6-Cryptography.md]: UUID seems the more generic so I think I prefer that

**Review by tghosth — APPROVED** [review]

**Pull request merged**